presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java

```diff
@@ -208,6 +208,12 @@ private PartialRow buildPrimaryKey()
     @Override
     public void close()
     {
+        try {
+            scanner.close();
```

Comment:
How did you find this?

Reply:
Our Kudu tserver's memory is all consumed and in the heap sampled heap usage diagram indicates they are consumed by row projector code gen related to scanner. And in Kudu logs, there are a lot of scanner idle expiring.